sdcm/nemesis.py

```diff
@@ -365,6 +364,19 @@ def disrupt_nodetool_cleanup(self):
         cmd = 'nodetool -h localhost cleanup keyspace1'
         self._run_nodetool(cmd, self.target_node)
 
+    def disrupt_modify_table_comment(self):
+        self._set_current_disruption('ModifyTableProperties %s' % self.target_node)
```

Comment:
IMHO string representation of node is more informative - it contains both addresses